diff --git a/lib/index.js b/lib/index.js
index a4e0170c34f03b3b91c3de8375287bd85e437a4e..bed4f2834795e1b94f0c8b0fca0225ba617cf68f 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -29,7 +29,6 @@ const subject_full_stop_1 = require("./subject-full-stop");
 const subject_max_length_1 = require("./subject-max-length");
 const subject_min_length_1 = require("./subject-min-length");
 const subject_exclamation_mark_1 = require("./subject-exclamation-mark");
-const trailer_exists_1 = require("./trailer-exists");
 const type_case_1 = require("./type-case");
 const type_empty_1 = require("./type-empty");
 const type_enum_1 = require("./type-enum");
@@ -65,7 +64,6 @@ exports.default = {
     'subject-max-length': subject_max_length_1.subjectMaxLength,
     'subject-min-length': subject_min_length_1.subjectMinLength,
     'subject-exclamation-mark': subject_exclamation_mark_1.subjectExclamationMark,
-    'trailer-exists': trailer_exists_1.trailerExists,
     'type-case': type_case_1.typeCase,
     'type-empty': type_empty_1.typeEmpty,
     'type-enum': type_enum_1.typeEnum,
