diff --git a/lib/index.js b/lib/index.js
index 69914a964c07863cbf241ac2e275e9abb564ba1a..d9413ec1de521a29d9e1672559fa69ca1877935c 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -29,7 +29,7 @@ import { subjectFullStop } from "./subject-full-stop.js";
 import { subjectMaxLength } from "./subject-max-length.js";
 import { subjectMinLength } from "./subject-min-length.js";
 import { subjectExclamationMark } from "./subject-exclamation-mark.js";
-import { trailerExists } from "./trailer-exists.js";
+// import { trailerExists } from "./trailer-exists.js";
 import { typeCase } from "./type-case.js";
 import { typeEmpty } from "./type-empty.js";
 import { typeEnum } from "./type-enum.js";
@@ -67,7 +67,7 @@ export default {
     "subject-full-stop": subjectFullStop,
     "subject-max-length": subjectMaxLength,
     "subject-min-length": subjectMinLength,
-    "trailer-exists": trailerExists,
+    // "trailer-exists": trailerExists,
     "type-case": typeCase,
     "type-empty": typeEmpty,
     "type-enum": typeEnum,
