diff --git a/writerOpts.js b/writerOpts.js
index 991947b85e2eac0e0ae246037624276882abb741..f133a294bd9ae17bb4f85c636923ecf3e69ba2c3 100644
--- a/writerOpts.js
+++ b/writerOpts.js
@@ -1,23 +1,10 @@
 'use strict'
 
 const compareFunc = require('compare-func')
-const { readFile } = require('fs').promises
-const { resolve } = require('path')
 
 async function createWriterOpts () {
-  const [template, header, commit, footer] = await Promise.all([
-    readFile(resolve(__dirname, './templates/template.hbs'), 'utf-8'),
-    readFile(resolve(__dirname, './templates/header.hbs'), 'utf-8'),
-    readFile(resolve(__dirname, './templates/commit.hbs'), 'utf-8'),
-    readFile(resolve(__dirname, './templates/footer.hbs'), 'utf-8')
-  ])
   const writerOpts = getWriterOpts()
 
-  writerOpts.mainTemplate = template
-  writerOpts.headerPartial = header
-  writerOpts.commitPartial = commit
-  writerOpts.footerPartial = footer
-
   return writerOpts
 }
 
